### PR TITLE
iPhone SE: Cannot add email address in contacts

### PR DESCRIPTION
**Pull request by freshking**
## Description

Adjust content inset when keyboard is visible

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5575

## Screenshots

Simulator Screen Shot - iPhone SE (1st generation) - 2021-03-12 at 09 24 14

### CODE REVIEW

**Comment by Ein-Tim** [comment]
When checking out this branch and testing on an simulator via xCode (iPhone SE, iOS 14.4), every time I try to enter something into the e-mail field the save button jumps above it.
Is this something with my environment/simulator? 
Please forgive me if this is normal on an emulator, I'm quite new with working with Xcode, etc.

<details>
<summary>Screen recording</summary>

https://user-images.githubusercontent.com/67682506/110853591-169c5380-82b4-11eb-807a-156fb59687ce.mov


</details>

**Comment by dsarkar** [comment]
Fixes https://github.com/corona-warn-app/cwa-app-ios/issues/2185

**Comment by freshking** [comment]
> When checking out this branch and testing on an simulator via Xcode (iPhone SE, iOS 14.4), every time I try to enter something into the e-mail field the save button jumps above it.
> Is this something with my environment/simulator?
> Please forgive me if this is normal on an emulator, I'm quite new with working with Xcode, etc.
> 
> Screen recording

@Ein-Tim Please re-test. I changed the VC to a UITableViewController which should now handle keyboard and scrolling a lot better without implementing hacky code.

**Comment by Ein-Tim** [comment]
Works as intended now @freshking. Thanks!

**Review by Ein-Tim — APPROVED** [review]

**Review by tcfos — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewController.swift [hunk 1]
```diff
@@ -46,31 +44,74 @@ class DiaryAddAndEditEntryViewController: UIViewController, UITextFieldDelegate,
...
+		}
+		guard let row = Row(rawValue: indexPath.row) else {
+			fatalError("Something went horribly wrong")
+		}
+		cell.textField.tag = row.rawValue
+		cell.textField.addTarget(self, action: #selector(textValueChanged(sender:)), for: .editingChanged)
+		cell.textField.delegate = self
+		switch row {
+		case .name:
+			cell.textField.accessibilityIdentifier = AccessibilityIdentifiers.ContactDiaryInformation.EditEntries.nameTextField
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewController.swift:78]: Thanks for adding IDs!

**Review by whiskey — APPROVED** [review]

**Review by janetback — APPROVED** [review]

**Pull request merged**